pantab/__init__.py

```diff
@@ -77,6 +77,11 @@ def _append_args_for_val_and_accessor(arg_l, val, accessor):
             # will lose precision compared to pandas type
             arg_l.append(val.microseconds // 100)
         """
+    elif accessor == 'setString':
+        if not val or not isinstance(val, str):
```

Comment:
Hmm I don't think this will work as it will probably drop non-str instances from an `object` column, so for example if you had data like [1, 2, "foo", "bar"] it would drop the first two values

Can you instead add a masking operation in `frame_to_hyper` that replaces missing values with an empty string for object dtypes?

Reply:
Sorry, I'm not much of a `pytest` person (I mostly use `nose`). And `pytest` keeps flooding me with deprecation warnings in my virtualenv (Py 3.7 with `pytest` 5.0.1).
Also, having said that, I'd have to see on how to generate a *simple* test case to trigger this situation. In my case it was through the Pandas read function for CSV or Parquet files. And I don't want to bloat the test with that.
I'll see if I've got time to tidy it up in some way, but I won't be (able) to look more deeply into doing stuff with `pytest` as that's outside the scope of what I can expend time on right now. Sorry ...